### DISCUSSION TITLE
Support for multiple priming doses

### DISCUSSION THREAD

**Discussion by floheigl** [NightscoutFoundation/xDrip#2512] [Ideas]
Hi,
Just started using NovoPen Echo and very happy with the support in xdrip!

One issue:
Sometimes I prime with 0.5 units and if i don't see enough of a drop coming out, I follow-up with a second one before doing the actual injection. (all within at most 20 seconds)
When I have just inserted a new cartridge, it might even take more than 2 priming doses before insulin starts coming out.

As far as I can tell (and read in the[ code](https://github.com/NightscoutFoundation/xDrip/blob/7352414170bf420b2572613c7e01b1b8728cebcd/app/src/main/java/com/eveningoutpost/dexdrip/insulin/opennov/data/SaveCompleted.java#L116)), xDrip doesn't cover that usecase and only "recognizes" the first priming dose as such.

I don't see a reason why we shouldn't consider ALL (however many) doses below priming-threshold and within the priming-time-window before a real dose as priming doses.

**Comment by tolot27** [comment]
@jamorham What do you think about this feature request? At the time I've used the NovoPen I run into the same issue and had to remove the additional "pseudo" treatments.
